src/test/regress/multi_schedule

```diff
@@ -80,9 +80,7 @@ test: multi_join_order_tpch_large
 # new shards before these tests, as they expect specific shard identifiers in
 # the output.
 # ----------
-test: multi_large_table_join_planning
-test: multi_large_table_pruning
-test: multi_large_table_task_assignment
+test: multi_large_table_join_planning multi_large_table_pruning multi_large_table_task_assignment
```

Comment:
these tests are read-only, so even if they use the same tables, they do not cause a race-condition.